gg-pac-team3-back/src/controllers/quizController.mjs

```diff
@@ -0,0 +1,114 @@
+import admin from '../config/firebase.mjs';
+import QuizModel from '../models/QuizModel.mjs';
+
+const db = admin.firestore();
+const QUIZZES = 'Quizzes';
+const COURSES = 'Courses';
+const LESSONS = 'lessons';
```

Comment:
ditto

Reply:
nit: since you're using these strings in multiple files, you might want to create a `Constants.mjs` or `Utils.mjs` file inside the `controllers` folder, and import it.

Reply:
Got it! will add Constants.mjs file